### PR TITLE
(FM-3866) Release Supported version 1.0.0

### PR DESCRIPTION
**Pull request by ferventcoder**
- [x] Sort the tickets into bug fixes and features and weed out the duplicates

### CODE REVIEW

**Comment by cowofevil** [comment]
@ferventcoder I thought metadata.json changes were always done via module-sync.

**Comment by ferventcoder** [comment]
@cowofevil not that I'm aware of. It's too different to do so.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,30 +1,91 @@
...
+(MODULES-2777) Build all DSC Types
+(MODULES-2520) Simplify type template / regenerate
+(MODULES-2520) Remove dscmeta_import_resource
+(MODULES-2777) Rebuild all DSC Types at 84a467c
+(MODULES-2837) Specify DSC Resource Module Path
+(MODULES-2641) Invoke-DscResource triggers reboot
+(MODULES-2799) Build custom types
+(MODULES-2512) Cannot ensure absent on Service DSC Resource
+(MODULES-2709) Reuse PowerShell instance for DSC
+(MODULES-2709) Disable Ruby 1.9.3 x86 shared PS
```
Comment (Iristyle) [inline CHANGELOG.md]: Added a PR about this change to the README at https://github.com/puppetlabs/puppetlabs-dsc/pull/174

Comment (ferventcoder) [inline CHANGELOG.md]: :+1:

**Comment by ferventcoder** [comment]
@puppetlabs/windows this is ready for review - compare to https://tickets.puppetlabs.com/browse/MODULES-2216

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,56 +1,83 @@
...
-###Summary
+First supported release! Performance is 2x+ by reusing PowerShell instance! Support PSCredentials, KeyValuePair and other Embedded Instance Classes. Update DSC Modules to latest available. Notify on Reboot required (and handle it with known named Puppet reboot resource).
+
+### Features
+
+- Reuse PowerShell instance for increased performance ([MODULES-2709](https://tickets.puppetlabs.com/browse/MODULES-2709))
+- Support PSCredentials, KeyValuePair and other Embedded Instance Classes ([MODULES-2178](https://tickets.puppetlabs.com/browse/MODULES-2178))
+- Remove LCM Refresh Mode Disabled Requirement ([MODULES-2575](https://tickets.puppetlabs.com/browse/MODULES-2575))
+- Include Supportable DSC Resource Types ([MODULES-2244](https://tickets.puppetlabs.com/browse/MODULES-2244))
+- Update DSC Modules to latest available (84a467c) and reduce the size by removing unnecessary files ([MODULES-2692](https://tickets.puppetlabs.com/browse/MODULES-2692)) ([MODULES-2777](https://tickets.puppetlabs.com/browse/MODULES-2777))
```
Comment (ferventcoder) [inline CHANGELOG.md]: This line is a bit of misnomer - we should probably spell out the exact date we are up to date with.

Comment (ferventcoder) [inline CHANGELOG.md]: And probably only include the link to the latest ticket.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,56 +1,83 @@
...
+
+- Fix: Module dependencies are incompatible with Puppet 3.x ([MODULES-2514](https://tickets.puppetlabs.com/browse/MODULES-2514))
+- Fix: Only allow PowerShell v5 ([MODULES-2521](https://tickets.puppetlabs.com/browse/MODULES-2521))
+- Fix: Setting LCM Mode to "Pull" Causes crash ([MODULES-2485](https://tickets.puppetlabs.com/browse/MODULES-2485))
+- Fix: Empty password for PSCredential causes crash ([MODULES-2615](https://tickets.puppetlabs.com/browse/MODULES-2615))
+- Fix: Module doesn't properly handle signed integers ([MODULES-2759](https://tickets.puppetlabs.com/browse/MODULES-2759))
+- Fix: Package resource fails when ReturnCode is specified ([MODULES-2562](https://tickets.puppetlabs.com/browse/MODULES-2562))
+- Fix: Symlinking vendored modules causes duplicate errors if the same version of one of the modules is already installed ([MODULES-2837](https://tickets.puppetlabs.com/browse/MODULES-2837))
+
+See any known issues or limitations in the ReadMe.
```
Comment (ferventcoder) [inline CHANGELOG.md]: Specifically NOT linking here.

File: CHANGELOG.md (hunk 2 above)
Comment (ferventcoder) [inline CHANGELOG.md]: @Iristyle @jpogran feedback on this bit?

Comment (jpogran) [inline CHANGELOG.md]: Could we link to the commit we did it? IIRC we put the commit sha link in the commit msg so they can link directly to where/when we got it from.

Comment (ferventcoder) [inline CHANGELOG.md]: Usually this is high level enough that they follow to the modules link and then find the sha from there.

Comment (ferventcoder) [inline CHANGELOG.md]: I have the SHA above right now

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,56 +1,83 @@
...
-###Summary
+First supported release! Performance is 2x+ by reusing PowerShell instance! Support PSCredentials, KeyValuePair and other Embedded Instance Classes. Update DSC Modules to latest available. Notify on Reboot required (and handle it with known named Puppet reboot resource).
+
+### Features
+
+- Reuse PowerShell instance for increased performance ([MODULES-2709](https://tickets.puppetlabs.com/browse/MODULES-2709))
+- Support PSCredentials, KeyValuePair and other Embedded Instance Classes ([MODULES-2178](https://tickets.puppetlabs.com/browse/MODULES-2178))
+- Remove LCM Refresh Mode Disabled Requirement ([MODULES-2575](https://tickets.puppetlabs.com/browse/MODULES-2575))
+- Include Supportable DSC Resource Types ([MODULES-2244](https://tickets.puppetlabs.com/browse/MODULES-2244))
+- Update DSC Modules to latest available as of November 15, 2015. (This is 84a467c of https://github.com/PowerShell/DscResources/) and reduce the size by removing unnecessary files ([MODULES-2777](https://tickets.puppetlabs.com/browse/MODULES-2777))
```
Comment (ferventcoder) [inline CHANGELOG.md]: @jpogran @puppetlabs/windows thoughts on this statement?

File: CHANGELOG.md (hunk 2 above)
Comment (ferventcoder) [inline CHANGELOG.md]: I updated this

**Comment by ferventcoder** [comment]
Well at least I got the branch name correct.

**Pull request closed without merging**